File: mychatApp-master/mychatApp-master/src/app/Model/UserLogin.ts
export class UserLogin {
  constructor() {
  }
  public id;
  public name: string;
  public password: string;
  public ConnectionID: string;
  public isConnect: string;
  public countmsg: number = 0;

}
export class Messages {
  constructor() {}
  public id;
  public sender: string;
  public recevier: string;
  public message: string;
  public time: Date;
  public IsRead: boolean;
}

File: mychatApp-master/mychatApp-master/Models/UserLogin.cs
using System;

namespace ChatApplication.Models
{
  public class UserLogin
    {
    public int id { get; set; }
    public string name { get; set; }
    public string password { get; set; }
    public string ConnectionID { get; set; }
    public string isConnect { get; set; }

  }
  public class Messages
  {
    public int id { get; set; }
    public string sender { get; set; }
    public string recevier { get; set; }
    public string message { get; set; }
    public DateTime time { get; set; }
    public bool IsRead { get; set; }
  }
}
